pkg/state/steps_shared.go

```diff
@@ -466,6 +466,14 @@ func (s *State) EnableSysExtensions(g *herd.Graph, opts ...herd.OpOption) error
 					}
 				}
 				// it has to link to the final dir after initramfs, so we avoid setting s.path here for the target
```

Comment:
This comment should stay close wo the `Symlink` line below?

Reply:
ahh crap